applications/espresso.yml

```diff
@@ -0,0 +1,93 @@
+name: espresso
+shortName: qe
+summary: Quantum Espresso
+defaultVersion: "5.4.0"
+versions:
+  # version 5.2.1
+  - version: "5.2.1"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  - version: "5.2.1"
+    isDefault: false
+    build: GNU
+    hasAdvancedComputeOptions: true
+  - version: "5.2.1"
+    isDefault: false
+    build: Intel
+    hasAdvancedComputeOptions: true
+  # version 5.4.0
+  - version: "5.4.0"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  - version: "5.4.0"
+    isDefault: false
+    build: GNU
+    hasAdvancedComputeOptions: true
+  - version: "5.4.0"
+    isDefault: false
+    build: Intel
+    hasAdvancedComputeOptions: true
+  # version 6.0.0
+  - version: "6.0.0"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  - version: "6.0.0"
+    isDefault: false
+    build: GNU
+    hasAdvancedComputeOptions: true
+  - version: "6.0.0"
+    isDefault: false
+    build: Intel
+    hasAdvancedComputeOptions: true
+  # version 6.3
+  - version: "6.3"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  - version: "6.3"
+    isDefault: false
+    build: GNU
+    hasAdvancedComputeOptions: true
+  - version: "6.3"
+    isDefault: false
+    build: Intel
+    hasAdvancedComputeOptions: true
+  # version 6.4.1
+  - version: "6.4.1"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  # version 6.5.0
+  - version: "6.5.0"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  # version 6.6.0
+  - version: "6.6.0"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  # version 6.7.0
+  - version: "6.7.0"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  # version 6.8.0
```

Comment:
maybe `#### version 6.8.0 ####` or something like that to make it more pronounced